### PR TITLE
@bentley/imodeljs-native 3.5.0

### PR DESCRIPTION
**Pull request by nick4598**

### CODE REVIEW

**Review by pmconne — APPROVED** [review]

**Comment by nick4598** [comment]
@pmconne 

Who is responsible for the @itwin/components-react package?
<details>
<summary>--[ FAILURE: @itwin/components-react ]--------------[ 1 minute 47.6 seconds ]--</summary>

undefined
Warning: componentWillReceiveProps has been renamed, and is not recommended for use. See https://reactjs.org/link/unsafe-component-lifecycles for details.

* Move data fetching code or side effects to componentDidUpdate.
* If you're updating state whenever props change, refactor your code to use memoization techniques or move it to static getDerivedStateFromProps. Learn more at: https://reactjs.org/link/derived-state
* Rename componentWillReceiveProps to UNSAFE_componentWillReceiveProps to suppress this warning in non-strict mode. In React 18.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: ReactDataGrid, Viewport, t
Warning: componentWillReceiveProps has been renamed, and is not recommended for use. See https://reactjs.org/link/unsafe-component-lifecycles for details.

* Move data fetching code or side effects to componentDidUpdate.
* If you're updating state whenever props change, refactor your code to use memoization techniques or move it to static getDerivedStateFromProps. Learn more at: https://reactjs.org/link/derived-state
* Rename componentWillReceiveProps to UNSAFE_componentWillReceiveProps to suppress this warning in non-strict mode. In React 18.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: t
ERROR: Coverage for branches (99.88%) does not meet global threshold (99.9%)
</details>

**Comment by pmconne** [comment]
> ERROR: Coverage for branches (99.88%) does not meet global threshold (99.9%)

@saskliutas

**Comment by aruniverse** [comment]
@nick4598 see [pr](https://github.com/iTwin/itwinjs-core/pull/4405)

**Review by aruniverse — APPROVED** [review]
going to approve on behalf of the mobile team; ci succeeds (builds ios dta ts +swift w/ new native)

**Comment by pmconne** [comment]
@nick4598 just decrease the threshold in package.json for now.

**Comment by nick4598** [comment]
decreased threshold to 99.8

File: common/changes/@itwin/components-react/native-3.5.0_2022-09-30-14-19.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/components-react",
+      "comment": "decrease threshold to 99.8 for test coverage",
```
Comment (aruniverse) [inline common/changes/@itwin/components-react/native-3.5.0_2022-09-30-14-19.json:5]:
```suggestion
      "comment": "",
```

**Comment by pmconne** [comment]
@calebmshafer @wgoehrig is some manual intervention required to get OIDC scopes working in QA?

**Review by calebmshafer — APPROVED** [review]

**Pull request merged**